app/data_models/answer_store.py

```diff
@@ -134,3 +136,27 @@ def remove_all_answers_for_list_item_id(self, list_item_id: str):
 
     def serialize(self):
         return list(self.answer_map.values())
+
+    def get_escaped_answer(self, answer_id, list_item_id):
+        if answer := self.get_answer(answer_id, list_item_id):
+            if isinstance(answer.value, list):
+                escaped_list = []
+                for list_item in answer.value:
+                    escaped_list.append(escape(list_item)) if isinstance(
+                        list_item, str
+                    ) else escaped_list.append(list_item)
+                return escaped_list
```

Comment:
The original version used list comprehension to avoid the need for a variable:
```suggestion
                return [
                    escape(list_item)
                    if list_item and isinstance(list_item, str)
                    else answer.value
                    for list_item in answer.value
                ]
```

Reply:
Done, thanks!